server-phoenix/priv/static/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>Helios2</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+  </head>
+
+  <body>
+    <link rel="stylesheet" type="text/css" href="./assets/application.css" />
+    <script type="module" src="assets/application.js"></script>
+  </body>
+</html>
```

Comment:
You could also mock out `File.read!("priv/static/index.html")` to give you a string literal that you assert. IIRC, Billy either merged or has a PR that mocks an http rest request for slack.